specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/EasyAuthConfigs.json

```diff
@@ -0,0 +1,706 @@
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Facebook": {
+      "description": "The configuration settings of the Facebook provider.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "<code>false</code> if the Facebook provider should not be enabled despite the set registration; otherwise, <code>true</code>.",
+          "type": "boolean"
+        },
+        "registration": {
+          "$ref": "#/definitions/AppRegistration",
+          "description": "The configuration settings of the app registration for the Facebook provider."
+        },
+        "graphApiVersion": {
+          "description": "The version of the Facebook api to be used while logging in.",
+          "type": "string"
+        },
+        "login": {
+          "$ref": "#/definitions/LoginScopes",
+          "description": "The configuration settings of the login flow."
+        }
+      }
+    },
+    "AppRegistration": {
+      "description": "The configuration settings of the app registration for providers that have app ids and app secrets",
+      "type": "object",
+      "properties": {
+        "appId": {
+          "description": "The App ID of the app used for login.",
+          "type": "string"
+        },
+        "appSecretRefName": {
+          "description": "The app secret ref name that contains the app secret.",
+          "type": "string"
+        }
+      }
+    },
+    "LoginScopes": {
+      "description": "The configuration settings of the login flow, including the scopes that should be requested.",
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "description": "A list of the scopes that should be requested while authenticating.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "GitHub": {
+      "description": "The configuration settings of the GitHub provider.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "<code>false</code> if the GitHub provider should not be enabled despite the set registration; otherwise, <code>true</code>.",
+          "type": "boolean"
+        },
+        "registration": {
+          "$ref": "#/definitions/ClientRegistration",
+          "description": "The configuration settings of the app registration for the GitHub provider."
+        },
+        "login": {
+          "$ref": "#/definitions/LoginScopes",
+          "description": "The configuration settings of the login flow."
+        }
+      }
+    },
+    "ClientRegistration": {
+      "description": "The configuration settings of the app registration for providers that have client ids and client secrets",
+      "type": "object",
+      "properties": {
+        "clientId": {
+          "description": "The Client ID of the app used for login.",
+          "type": "string"
+        },
+        "clientSecretRefName": {
```

Comment:
>clientSecretRefName [](http://example.com/codeflow?start=9&length=19)

Add x-ms-secret extension if this is a secret.

Reply:
Not a secret